<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>occQtClass</class>
 <widget class="QMainWindow" name="occQtClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SimpleGM</string>
  </property>
  <property name="windowIcon">
   <iconset resource="occqt.qrc">
    <normaloff>:/Resources/lamp.png</normaloff>:/Resources/lamp.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>33</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <addaction name="actionZoom"/>
    <addaction name="actionPan"/>
    <addaction name="actionRotate"/>
    <addaction name="separator"/>
    <addaction name="actionReset"/>
    <addaction name="actionFitAll"/>
    <addaction name="actionDeleteAll"/>
   </widget>
   <widget class="QMenu" name="menuPrimitive">
    <property name="title">
     <string>Primi&amp;tive</string>
    </property>
    <addaction name="actionBox"/>
    <addaction name="actionCone"/>
   </widget>
   <widget class="QMenu" name="menuModeling">
    <property name="title">
     <string>3D</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="separator"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menu2D">
    <property name="title">
     <string>2D</string>
    </property>
    <addaction name="actionPoints"/>
    <addaction name="actionLines"/>
    <addaction name="actionSplines"/>
    <addaction name="actionBspline"/>
   </widget>
   <widget class="QMenu" name="menuSurface">
    <property name="title">
     <string>Surfaces</string>
    </property>
    <addaction name="actionbezier_surface"/>
    <addaction name="actionsurface_2boundares"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
   <addaction name="menuPrimitive"/>
   <addaction name="menu2D"/>
   <addaction name="menuSurface"/>
   <addaction name="menuModeling"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionExit">
   <property name="text">
    <string>&amp;Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionZoom">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="occqt.qrc">
     <normaloff>:/Resources/Zoom.png</normaloff>:/Resources/Zoom.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Zoom</string>
   </property>
  </action>
  <action name="actionPan">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="occqt.qrc">
     <normaloff>:/Resources/Pan.png</normaloff>:/Resources/Pan.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Pan</string>
   </property>
  </action>
  <action name="actionRotate">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="occqt.qrc">
     <normaloff>:/Resources/Rotate.png</normaloff>:/Resources/Rotate.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Rotate</string>
   </property>
  </action>
  <action name="actionReset">
   <property name="icon">
    <iconset resource="occqt.qrc">
     <normaloff>:/Resources/Home.png</normaloff>:/Resources/Home.png</iconset>
   </property>
   <property name="text">
    <string>&amp;BaseView</string>
   </property>
  </action>
  <action name="actionFitAll">
   <property name="icon">
    <iconset resource="occqt.qrc">
     <normaloff>:/Resources/FitAll.png</normaloff>:/Resources/FitAll.png</iconset>
   </property>
   <property name="text">
    <string>&amp;FitAll</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="occqt.qrc">
     <normaloff>:/Resources/lamp.png</normaloff>:/Resources/lamp.png</iconset>
   </property>
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="actionBox">
   <property name="icon">
    <iconset resource="occqt.qrc">
     <normaloff>:/Resources/box.png</normaloff>:/Resources/box.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Box</string>
   </property>
  </action>
  <action name="actionCone">
   <property name="icon">
    <iconset resource="occqt.qrc">
     <normaloff>:/Resources/cone.png</normaloff>:/Resources/cone.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Cone</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="occqt.qrc">
     <normaloff>:/Resources/fillet.png</normaloff>:/Resources/fillet.png</iconset>
   </property>
   <property name="text">
    <string>&amp;NewElement</string>
   </property>
  </action>
  <action name="actionDeleteAll">
   <property name="icon">
    <iconset resource="occqt.qrc">
     <normaloff>:/Resources/delete.PNG</normaloff>:/Resources/delete.PNG</iconset>
   </property>
   <property name="text">
    <string>&amp;DeleteAll</string>
   </property>
  </action>
  <action name="actionPoints">
   <property name="icon">
    <iconset resource="occqt.qrc">
     <normaloff>:/Resources/points.png</normaloff>:/Resources/points.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Points</string>
   </property>
  </action>
  <action name="actionLines">
   <property name="icon">
    <iconset resource="occqt.qrc">
     <normaloff>:/Resources/lines.png</normaloff>:/Resources/lines.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Lines</string>
   </property>
  </action>
  <action name="actionCircle">
   <property name="text">
    <string>&amp;Circle</string>
   </property>
  </action>
  <action name="actionSplines">
   <property name="icon">
    <iconset resource="occqt.qrc">
     <normaloff>:/Resources/spline.png</normaloff>:/Resources/spline.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Splines</string>
   </property>
  </action>
  <action name="actionBspline">
   <property name="icon">
    <iconset resource="occqt.qrc">
     <normaloff>:/Resources/bspline.png</normaloff>:/Resources/bspline.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Bspline</string>
   </property>
   <property name="toolTip">
    <string>Bspline</string>
   </property>
  </action>
  <action name="actionbezier_surface">
   <property name="icon">
    <iconset resource="occqt.qrc">
     <normaloff>:/Resources/bezier.png</normaloff>:/Resources/bezier.png</iconset>
   </property>
   <property name="text">
    <string>&amp;bezier_surface</string>
   </property>
  </action>
  <action name="actionsurface_2boundares">
   <property name="icon">
    <iconset resource="occqt.qrc">
     <normaloff>:/Resources/some_surface3.png</normaloff>:/Resources/some_surface3.png</iconset>
   </property>
   <property name="text">
    <string>&amp;surface_2boundares</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="occqt.qrc"/>
 </resources>
 <connections/>
</ui>
